iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -321,6 +323,16 @@ public void configureJobConf(TableDesc tableDesc, JobConf jobConf) {
       jobConf.set(opKey, tableDesc.getProperties().getProperty(opKey));
       Preconditions.checkArgument(!tableName.contains(TABLE_NAME_SEPARATOR),
           "Can not handle table " + tableName + ". Its name contains '" + TABLE_NAME_SEPARATOR + "'");
+      if (tableDesc.getProperties()
+          .get(HiveCustomStorageHandlerUtils.WRITE_OPERATION_CONFIG_PREFIX + tableName) != null) {
+        jobConf.set(HiveCustomStorageHandlerUtils.WRITE_OPERATION_CONFIG_PREFIX + tableName, tableDesc.getProperties()
+            .getProperty(HiveCustomStorageHandlerUtils.WRITE_OPERATION_CONFIG_PREFIX + tableName));
+      }
+      if (tableDesc.getProperties()
+              .get(HiveCustomStorageHandlerUtils.WRITE_OPERATION_FOR_MERGE_TASK + tableName) != null) {
```

Comment:
you can create utils method like `setMergeTaskWriteOperation` & `getMergeTaskWriteOperation `

Reply:
Added util methods in TableDesc to get and set property.